src/requirements.txt

```diff
@@ -32,4 +32,4 @@ requests==2.25.1
 # git+https://github.com/hubmapconsortium/commons.git@${COMMONS_BRANCH}#egg=hubmap-commons
 hubmap-commons==2.0.14
 
-
+boto3
```

Comment:
We should specify the actual version number. 1.24.47 is their latest release: https://pypi.org/project/boto3/#history

Reply:
Okay.  I wasn't sure what the best practice was for this, which I would guess would be tied to [my comment on the preceding conversation](https://github.com/hubmapconsortium/uuid-api/pull/107#discussion_r941451982)

Reply:
Using a pinned version of boto3 just so when we build the docker image, we don't pull their new updates that may cause any side effects if any. Looking at their release history, they have a very active/frequent release cycle.

Reply:
Code change tested locally and pushed.